uplift_report.ipynb

```diff
@@ -1,381 +1,417 @@
 {
-  "nbformat": 4,
-  "nbformat_minor": 0,
-  "metadata": {
-    "colab": {
-      "name": "uplift_report.ipynb",
-      "version": "0.3.2",
-      "provenance": [],
-      "collapsed_sections": [],
-      "toc_visible": true,
-      "include_colab_link": true
-    },
-    "kernelspec": {
-      "name": "python3",
-      "display_name": "Python 3"
-    }
+ "cells": [
```

Comment:
The entire notebook is changed, possibly because it was edited not in Google Colab.

While our Jupyter is _mostly_ compatible, it's not identical, and subsequent PRs would be hard to analyze, because they, made from Colab, would change the entire notebook again.

Please, revert this change and do minimal changes via Colab (or, honestly, a text editor is sometimes enough to add/modify a line or two), and then commit from there.

Reply:
👍